docs/jwst/master_background/description.rst

```diff
@@ -338,7 +338,7 @@ During :ref:`calwebb_spec2 <calwebb_spec2>` processing, all source and backgroun
 slits are first partially calibrated up through the :ref:`extract_2d <extract_2d_step>`
 and :ref:`srctype <srctype_step>` steps of :ref:`calwebb_spec2 <calwebb_spec2>`,
 which results in 2D cutouts for each slit with the source type identified. At this
-point the `master_background_nrs_slits` step is applied, which is a unique version
+point the `master_background_mos_step` step is applied, which is a unique version
```

Comment:
don't need to include "_step" in the name here, do we? We didn't include that suffix in the old name.

Reply:
✅